tests/integration/telemetry/stats/prometheus/nullvm/customize_metrics_test.go

```diff
@@ -0,0 +1,129 @@
+// +build integ
```

Comment:
Is it possible to move the original test to here, so that update to it could shown clearly on github.

Reply:
I actually did the move like the dashboard test but somehow github took it as a copy paste :(